Tesla.css

```diff
@@ -0,0 +1,477 @@
+.title__h2 {
+  margin: 0 auto;
+  font-size: 15px;
+  font-weight: 500;
+  text-align: center;
+}
+
+.image__car {
+  display: block;
+  size: contain;
+  width: 67%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 26px;
+}
+
+.box {
+  margin: 0 auto;
+  margin-bottom: 37px;
+  max-width: 800px;
+  width: 56%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.box__item {
+  display: flex;
+  align-items: center;
+}
+
+.box__img-1 {
+  width: 46px;
+  height: 34.13px;
+  margin-right: 14px;
+  padding: 0;
+}
+
+.box__img-2 {
+  width: 30px;
+  height: 50px;
+  margin-right: 11px;
+}
+
+.test__img-3 {
+  width: 22px;
+  height: 22px;
+}
+
+.box__item-1 {
+  margin-right: 83px;
+}
+
+.box__item-2 {
+  margin-right: 105px;
+}
+
+.box__text{
+  display: inline-block;
+  font-weight: 600;
+}
+
+.box__text-speed {
+  font-size: 12px;
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: normal;
+  margin: 0;
+}
+
+.box__text-number {
+  font-weight: 600;
+  font-size: 35px;
+  margin: 0;
+}
+
+.box__text-sec {
+  font-size: 20px;
+  margin-left: 6px;
+}
+
+.mobile-box {
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+}
+
+.mobile-test {
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+}
+
+.test {
+  margin-right: 40px;
+  display: flex;
+  align-items: center;
+}
+
+.test__img-3 {
+  margin-top: 20px;
+  margin-bottom: 16px;
+  margin-right: 7px;
+  width: 22px;
+  height: 22px;
+}
+
+.test__text {
+  margin: 0;
+  margin-top: 23px;
+  margin-bottom: 17px;
+  text-transform: uppercase;
+  font-size: 13px;
+  font-weight: bold;
+  white-space: nowrap;
+}
+
+.vector {
+  width: 28px;
+  margin: 0 auto 93px;
+}
+
+@media screen and (max-width: 1200px) {
+  .nav__model {
+      display: none;
+  }
+  .box {
+      flex-wrap: wrap;
+  }
+  .mobile-box {
+      margin-bottom: 40px;
+  }
+}
+
+.main {
+    max-width: 1920px;
+}
+
+.future {
+  display: flex;
+}
+
+.future__h1 {
+  width: 69%;
+  font-size: 70px;
+  font-weight: 600;
+  color: #E51937;
+  margin: 0;
+  padding-left: 80px;
+  background: url(img/future.png) no-repeat;
+  background-size: cover;
+}
+
+.section {
+  background: #E51937;
+  color: #ffffff;
+}
+
+.section__future {
+  width: 31%;
+}
+
+.section__h1 {
+  margin: 50px 0 10px 51px;
+  font-weight: 600;
+  font-size: 30px;
+  white-space: nowrap;
+}
+
+.section__p {
+  font-size: 15px;
+  font-weight: 500;
+  margin: 0 72px 54px 52px;
+}
+
+.section__icon {
+    margin: 0 123px 171px 48px;
+}
+
+.item__icon {
+  display: flex;
+  align-items: center;
+  margin-bottom: 52px;
+}
+
+.icon__img {
+  margin-right: 20px;
+}
+
+.icon__img_bag {
+  margin-right: 24px;
+}
+
+.icon__text_b {
+  display: block;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.icon__text_s {
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.icon_number {
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 43px;
+}
+
+.link__section {
+  color: #FFF;
```

Comment:
Codacy found an issue: [Expected "#FFF" to be "#fff" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)